### DISCUSSION TITLE
When I create a point to point constrain, the bodies no longer collide with each other. Is this intended?

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#148] [Help / Questions] [answered]
I was making a ragdoll, but the parts of it I link with constraints no longer register collisions with each other. Is there a way to disable this? Or is it just a feature of ammo.js?

**Comment by yandeu** [comment, marked as answer]
Check the last param of each constraint called `disableCollisionsBetweenLinkedBodies`.

See https://github.com/enable3d/enable3d/blob/master/packages/ammoPhysics/src/constraints.ts

**Reply by N8python** [reply]
Thanks!
